geomesa-convert/geomesa-convert-xml/pom.xml

```diff
@@ -19,6 +19,12 @@
     <name>GeoMesa Convert XML</name>
 
     <dependencies>
+<dependency>
+    <groupId>xalan</groupId>
+    <artifactId>xalan</artifactId>
+    <version>2.7.2</version>
```

Comment:
Is this dependency required?  If so, we can piggyback on https://dev.eclipse.org/ipzilla/show_bug.cgi?id=9587.